components/Textarea/Textarea.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { Textarea } from './Textarea';
+import React from 'react';
+
+describe('Textarea Component', () => {
+  it('renders without crashing', () => {
+    render(<Textarea />);
+    const textarea = screen.getByTestId('textarea');
+    expect(textarea).toBeInTheDocument();
+  });
+
+  it('applies the provided className', () => {
+    const className = 'custom-class';
```

Comment:
You're only using this variable twice, does it make sense to initialize a variable for the value?

Reply:
I don't _need_ to but I didn't think it would hurt.